scripts/bootstrap

```diff
@@ -33,7 +35,7 @@ SECRET_PATH="."
 CONFIG_PATH="."
 ADD_STARTUP_SCRIPT=0
 
-while getopts 'bc:d:ghk:p:s:xC' c
+while getopts 'bc:d:ghk:p:s:xCod' c
```

Comment:
there's already a `d:`, the ending `d` shouldn't be there

Reply:
will fix